src/analysis/locally_convex/with_seminorms.lean

```diff
@@ -14,15 +14,23 @@ import analysis.locally_convex.bounded
 
 * `seminorm_family.basis_sets`: The set of open seminorm balls for a family of seminorms.
 * `seminorm_family.module_filter_basis`: A module filter basis formed by the open balls.
-* `seminorm.is_bounded`: A linear map `f : E →ₗ[𝕜] F` is bounded iff every seminorm in `F` can be
-bounded by a finite number of seminorms in `E`.
 
 ## Main statements
 
-* `continuous_from_bounded`: A bounded linear map `f : E →ₗ[𝕜] F` is continuous.
```

Comment:
I'm not sure why you're removing these from the module docstring given that you aren't removing them from this file (If they stay, the second one should be fixed to `seminorm.continuous_from_bounded` though).

Can you simplify the proofs of these with your new lemmas?

Reply:
These lemmas should not be used, lots of repeating code and the definition for boundedness is not the correct one. I can leave the docs in for now, though. It is not really relevant for this PR